requirements.txt

```diff
@@ -1,11 +1,25 @@
+blinker==1.7.0
```

Comment:
do we use all of these libraries? Let's take out the ones we don't use

Reply:
There are a lot of libraries here now, also curious if we use them all